.github/workflows/deploy_environment.yml

```diff
@@ -53,6 +53,7 @@ jobs:
           dotnet user-secrets set "trams:api_key" "${{ secrets.TRAMS_API_KEY }}"
           dotnet user-secrets set "app:username" "${{ secrets.APP_USERNAME }}"
           dotnet user-secrets set "app:password" "${{ secrets.APP_PASSWORD }}"
+          dotnet user-secrets set "AzureAd:ClientSecret" "${{ secrets.AZURE_AD_CLIENT_SECRET }}"
```

Comment:
I think we shouldn't be using user-secrets in deployments. As suggested, Chris Wright may be able to advise on the best approach, but my feeling is environment variables are the way to go

Reply:
We can have them in either for now - Until we get it fully containerised
It would probably be best to test both approaches, as I believe a lot of people will still prefer to develop on it running natively on their mahines, rather than in Docker

Reply:
Can I get this approved for now then ?

Reply:
@chrisdexnimble  and @Stretch96 really really sorry but I don't think we should be using user-secrets outside of dev. I'm right in thinking this new key would give access to our active directory? Feels like too big a risk to me.https://learn.microsoft.com/en-us/dotnet/architecture/microservices/secure-net-microservices-web-applications/developer-app-secrets-storage